strawberryfields/api/job.py

```diff
@@ -73,13 +73,15 @@ class Job:
         status (strawberryfields.api.JobStatus): the job status
         connection (strawberryfields.api.Connection): the connection over which the
             job is managed
+        meta (dict[str, str]): metadata related to job execution
     """
 
-    def __init__(self, id_: str, status: JobStatus, connection):
+    def __init__(self, id_: str, status: JobStatus, connection, meta: dict = None):
         self._id = id_
         self._status = status
         self._connection = connection
         self._result = None
+        self._meta = meta if meta is not None else {}
```

Comment:
Any reason for not using the syntax below instead?

```suggestion
        self._meta = meta or {}
```

Reply:
No reason, just didn't think of it :slightly_smiling_face: